src/vcpkg/binarycaching.cpp

```diff
@@ -2442,118 +2441,13 @@ std::string vcpkg::generate_nuspec(const Path& package_dir,
-               "NuGet CLI.");
-    tbl.format("x-azblob,<url>,<sas>[,<rw>]",
-               "**Experimental: will change or be removed without warning** Adds an Azure Blob Storage source. Uses "
-               "Shared Access Signature validation. URL should include the container path.");
-    tbl.format("x-gcs,<prefix>[,<rw>]",
-               "**Experimental: will change or be removed without warning** Adds a Google Cloud Storage (GCS) source. "
-               "Uses the gsutil CLI for uploads and downloads. Prefix should include the gs:// scheme and be suffixed "
-               "with a `/`.");
-    tbl.format("x-aws,<prefix>[,<rw>]",
-               "**Experimental: will change or be removed without warning** Adds an AWS S3 source. "
-               "Uses the aws CLI for uploads and downloads. Prefix should include s3:// scheme and be suffixed "
-               "with a `/`.");
-    tbl.format(
-        "x-aws-config,<parameter>",
-        "**Experimental: will change or be removed without warning** Adds an AWS S3 source. "
-        "Adds an AWS configuration; currently supports only 'no-sign-request' parameter that is an equivalent to the "
-        "'--no-sign-request parameter of the AWS cli.");
-    tbl.format("x-cos,<prefix>[,<rw>]",
-               "**Experimental: will change or be removed without warning** Adds an COS source. "
-               "Uses the cos CLI for uploads and downloads. Prefix should include cos:// scheme and be suffixed "
-               "with a `/`.");
-    tbl.format("interactive", "Enables interactive credential management for some source types");
-    tbl.blank();
-    tbl.text("The `<rw>` optional parameter for certain strings controls whether they will be consulted for "
-             "downloading binaries and whether on-demand builds will be uploaded to that remote. It can be specified "
-             "as 'read', 'write', or 'readwrite'.");
-    tbl.blank();
-    tbl.text("The `nuget` and `nugetconfig` source providers additionally respect certain environment variables while "
-             "generating nuget packages. The `metadata.repository` field will be optionally generated like:\n"
-             "\n"
-             "    <repository type=\"git\" url=\"$VCPKG_NUGET_REPOSITORY\"/>\n"
-             "or\n"
-             "    <repository type=\"git\"\n"
-             "                url=\"${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}.git\"\n"
-             "                branch=\"${GITHUB_REF}\"\n"
-             "                commit=\"${GITHUB_SHA}\"/>\n"
-             "\n"
-             "if the appropriate environment variables are defined and non-empty.\n");
-    tbl.blank();
-    tbl.text("NuGet's cache is not used by default. To use it for every nuget-based source, set the environment "
-             "variable `VCPKG_USE_NUGET_CACHE` to `true` (case-insensitive) or `1`.\n");
-    tbl.blank();
-    print2(tbl.m_str);
```

Comment:
I wish we didn't lose the nice formatting here :(
However, I don't think there is another way to have nice formatting and provide the necessary context to the localizers. 
FYI: Sorry I missed this!